app/src/main/res/layout/ongoing_story_card.xml

```diff
@@ -14,25 +14,26 @@
   <com.google.android.material.card.MaterialCardView
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:layout_marginStart="24dp"
-    android:layout_marginTop="8dp"
-    android:layout_marginEnd="24dp"
-    android:layout_marginBottom="8dp"
-    app:cardCornerRadius="2dp"
+    android:layout_marginStart="28dp"
+    android:layout_marginTop="28dp"
+    android:layout_marginEnd="28dp"
+    app:cardBackgroundColor="@color/white"
+    app:cardCornerRadius="4dp"
     app:cardElevation="4dp">
 
     <androidx.constraintlayout.widget.ConstraintLayout
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
-      android:onClick="@{viewModel::clickOnOngoingStoryTile}"
-      android:padding="8dp">
+      android:onClick="@{viewModel::clickOnOngoingStoryTile}">
 
       <ImageView
         android:id="@+id/lesson_thumbnail"
         android:layout_width="80dp"
-        android:layout_height="45dp"
+        android:layout_height="44dp"
+        android:layout_marginStart="8dp"
         android:importantForAccessibility="no"
         android:paddingStart="40dp"
+        android:paddingTop="4dp"
```

Comment:
Padding top and bottom is 22dp and padding start is 8dp. I think you can remove marginStart

Reply:
Actually, this padding is not related to card, instead this is related to image padding. Also, I have not defined the height of card, because if the learner increases the text size from settings, than it would mean that the text will actually go out of card on either top or bottom side, which we do not want.